style.css

```diff
@@ -887,6 +1095,19 @@ body {
     padding: 5px;
   }
 
+  .card-contentP h2 {
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 45px;
+    color: #091e42;
+    margin: 12px;
+  }
+
+  .card-contentP {
+    width: 100%;
+    height: 40%;
+  }
+
```

Comment:
- [ ] Nice work implementing most of the required changes, you are almost there. Kudos 👏. It would be nice for you to design your popup to match that of Figma's design. The _**technologies**_ and _**buttons**_ are on the bottom-right part of the pop while the _**description**_ is aligned to the bottom-left part of the popup. Kindly make your popup identical to that of Figma's template. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/158038891-4bfde8e8-f800-418e-b3a7-43fec59121b0.png)

### Figma's Design

![image](https://user-images.githubusercontent.com/77106962/158038922-8328cc80-1960-4937-babf-6d8d077ec14b.png)